src/view/restore-seed.js

```diff
@@ -52,7 +52,7 @@ const RestoreSeedView = ({ store, wallet }) => (
               wallet.setRestoreSeed({ word, index: seedIndex - 1 })
             }
             key={i}
-            autoFocus={i === 0}
+            autoFocus={i === 0 && store.wallet.autofocusRestoreInput}
```

Comment:
@valentinewallace Not really sure I understand ... why does `store.wallet.autofocusRestoreInput` need to be false for the `setRestoreSeed()` case?

Reply:
Disclaimer: I don't fully know why this solution works.

It needs `autofocusRestoreInput` to flip from false to true when the user turns the page, or else the bug persists. I decided to set it to false in `setRestoreSeed()` but there may be a better place.